### DISCUSSION TITLE
Including login_link in SMS text

### DISCUSSION THREAD

**Discussion by alpkaanaksu** [rubenarslan/formr.org#527] [Q&A] [answered]
We want to create a run and send daily reminders **via SMS** to the participants.

The email implementation does it automatically, if you include `{{login_link}}` or `{{login_code}}` in the text. We would like to do the same using the `text_message_twilio` function from the helper package. Is there a way to achieve this?

**Comment by rubenarslan** [comment, marked as answer]
https://github.com/rubenarslan/formr.org/wiki/How-to-send-text-messages-(SMS)
does this help?

**Reply by alpkaanaksu** [reply]
Yes, thank you
